media/transcoder/src/main/java/com/example/transcoder/CreateJobWithStaticOverlay.java

```diff
@@ -18,33 +18,41 @@
 
 // [START transcoder_create_job_with_static_overlay]
 
-import com.google.cloud.video.transcoder.v1beta1.AudioStream;
-import com.google.cloud.video.transcoder.v1beta1.CreateJobRequest;
-import com.google.cloud.video.transcoder.v1beta1.ElementaryStream;
-import com.google.cloud.video.transcoder.v1beta1.Input;
-import com.google.cloud.video.transcoder.v1beta1.Job;
-import com.google.cloud.video.transcoder.v1beta1.JobConfig;
-import com.google.cloud.video.transcoder.v1beta1.LocationName;
-import com.google.cloud.video.transcoder.v1beta1.MuxStream;
-import com.google.cloud.video.transcoder.v1beta1.Output;
-import com.google.cloud.video.transcoder.v1beta1.Overlay;
-import com.google.cloud.video.transcoder.v1beta1.Overlay.AnimationEnd;
-import com.google.cloud.video.transcoder.v1beta1.Overlay.AnimationStatic;
-import com.google.cloud.video.transcoder.v1beta1.Overlay.NormalizedCoordinate;
-import com.google.cloud.video.transcoder.v1beta1.TranscoderServiceClient;
-import com.google.cloud.video.transcoder.v1beta1.VideoStream;
+import com.google.cloud.video.transcoder.v1.AudioStream;
+import com.google.cloud.video.transcoder.v1.CreateJobRequest;
+import com.google.cloud.video.transcoder.v1.ElementaryStream;
+import com.google.cloud.video.transcoder.v1.Input;
+import com.google.cloud.video.transcoder.v1.Job;
+import com.google.cloud.video.transcoder.v1.JobConfig;
+import com.google.cloud.video.transcoder.v1.LocationName;
+import com.google.cloud.video.transcoder.v1.MuxStream;
+import com.google.cloud.video.transcoder.v1.Output;
+import com.google.cloud.video.transcoder.v1.Overlay;
+import com.google.cloud.video.transcoder.v1.Overlay.AnimationEnd;
+import com.google.cloud.video.transcoder.v1.Overlay.AnimationStatic;
+import com.google.cloud.video.transcoder.v1.Overlay.NormalizedCoordinate;
+import com.google.cloud.video.transcoder.v1.TranscoderServiceClient;
+import com.google.cloud.video.transcoder.v1.VideoStream;
 import com.google.protobuf.Duration;
 import java.io.IOException;
 
 public class CreateJobWithStaticOverlay {
 
   public static void main(String[] args) throws IOException {
     // TODO(developer): Replace these variables before running the sample.
-    String projectId = "my-project-id";
+    String projectId = "transcoder-test-288317";
     String location = "us-central1";
-    String inputUri = "gs://my-bucket/my-video-file";
-    String overlayImageUri = "gs://my-bucket/my-overlay-image.jpg"; // Must be a JPEG
-    String outputUri = "gs://my-bucket/my-output-folder/";
+    String templateId = "my-job-template";
+    String inputUri = "gs://nickcook-bucket/ChromeCast.mp4";
```

Comment:
Is this a publicly available bucket? If this is for demonstration purposes, we should move these test files into a public bucket.

Reply:
Fixed. These values were inadvertently left in from local testing.